raiden-dapp/src/components/transaction-history/Transaction.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="transaction">
+    <v-row class="transaction__item" no-gutters>
+      <v-col class="transaction__item__icon" cols="1">
+        <v-img
+          v-if="transfer.direction === 'sent'"
+          height="45px"
+          width="45px"
+          :src="require('../../assets/sent_transfer.svg')"
```

Comment:
```suggestion
          :src="require('@/assets/sent_transfer.svg')"
```
Apparenty `@` as root works with assets too.